python/hsfs/util.py

```diff
@@ -189,23 +189,34 @@ async def create_async_engine(
     external: bool,
```

Comment:
Maybe we should move that to its own file called async_mysql_client or something in the future